var/spack/repos/builtin/packages/kaldi/package.py

```diff
@@ -54,17 +56,15 @@ class Kaldi(Package):    # Does not use Autotools
     depends_on('sctk', type='run')
     depends_on('speex', type='run')
     depends_on('openfst@1.4.1-patch', when='@c024e8')
+    depends_on('openfst@1.6.0:', when='2018-07-11')
```

Comment:
Missing an `@` sign.

Reply:
that's corrected and i've updated the older version with the date it was committed